src/main/kotlin/com/openlattice/edm/tasks/EdmSyncInitializerTask.kt

```diff
@@ -0,0 +1,94 @@
+package com.openlattice.edm.tasks
+
+import com.openlattice.client.RetrofitFactory
+import com.openlattice.datastore.services.EdmManager
+import com.openlattice.edm.EdmApi
+import com.openlattice.edm.EntityDataModel
+import com.openlattice.tasks.HazelcastInitializationTask
+import com.openlattice.tasks.PostConstructInitializerTaskDependencies
+import com.openlattice.tasks.Task
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings
+import org.apache.olingo.commons.api.edm.FullQualifiedName
+import org.slf4j.LoggerFactory
+
+class EdmSyncInitializerTask : HazelcastInitializationTask<EdmSyncInitializerDependencies> {
+
+    companion object {
+        private val OL_AUDIT_FQN = FullQualifiedName("OPENLATTICE_AUDIT", "AUDIT")
```

Comment:
I think we should also include audit edge type here. And it would be nice to get the fqns from auditing.yaml.

Reply:
I don't think the actual audit entity types break the EDM import -- this one is legacy (we should probably just delete it from the EDM actually)

Reply:
Oh, I see. Totally confused this entity type / property types with current auditing. Yeah, the sync task works fine without removing these from edm.